content/400-reference/100-system-requirements.mdx

```diff
@@ -42,10 +42,12 @@ Prisma is supported on MacOS, Windows and most Linux distributions.
 Prisma requires the following system libraries installed to work:
 
 - OpenSSL 1.0.x, 1.1.x or 3.0.x
-- glibc 2.17 or newer
+- On glibc-based distributions, glibc 2.17 or newer (glibc 2.24 or newer on ARM64)
```

Comment:
What does "glibc-based distributions" really mean? Will all Linux users know what that means? Can that be made more concrete? 

> (glibc 2.24 or newer on ARM64)
So what are the other ones that are not "ARM64"?

Reply:
made a table instead, should now be more clear and most importantly more accurate